src/Fantomas.Tests/OperatorTests.fs

```diff
@@ -1226,3 +1226,40 @@ module Foo =
             | true -> id
             | false -> id)
 """
+
+let operator_application_literal_values =
```

Comment:
I took this list of literals from the [F# docs page](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/literals), but strictly speaking it's not necessary.  If we wanted to vary the behavior of the spacing then we'd use this to validate that.